addon/-private/system/ordered-set.js

```diff
@@ -16,6 +16,18 @@ OrderedSet.prototype = Object.create(EmberOrderedSet.prototype);
 OrderedSet.prototype.constructor = OrderedSet;
 OrderedSet.prototype._super$constructor = EmberOrderedSet;
 
+OrderedSet.prototype.pushMany = function pushMany(additions) {
+  for (let i = 0; i < additions.length; i++) {
+    let value = additions[i];
+
+    if (value) {
+      this.add(value);
+    }
+  }
+
+  return this.size;
+};
+
 OrderedSet.prototype.addWithIndex = function(obj, idx) {
   let guid = guidFor(obj);
   let presenceSet = this.presenceSet;
```

Comment:
This is unfortunately private API that will change. Ember-data shouldn't be reaching into private state, rather it should use `set.has`